### PR TITLE
Add Sezzle

### PR DESCRIPTION
**Pull request by Yang-Nankai**
Sezzle is a fintech platform that provides a "Buy Now, Pay Later" solution, enabling consumers to make purchases and split the payment into interest-free installments. It partners with merchants to offer flexible payment options that improve customer satisfaction and increase sales. Sezzle is widely used across e-commerce platforms, offering easy integration and user-friendly payment management tools. The platform offers 2FA for the account login verification process. The website is ranked 25,498 on [Similarweb](https://pro.similarweb.com/#/digitalsuite/websiteanalysis/overview/website-performance/*/999/1m?webSource=Total&key=sezzle.com). The website provides documentation on how to turn on 2FA.

### CODE REVIEW

**Comment by Yang-Nankai** [comment]
image
image

File: entries/s/sezzle.com.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "Sezzle": {
+    "domain": "sezzle.com",
+    "tfa": [
+      "sms"
+    ],
+    "notes": "Only merchants can enable 2FA.",
+    "documentation": "https://merchant-help.sezzle.com/hc/sections/16790628725780",
```
Comment (hkamran80) [inline entries/s/sezzle.com.json]:
```suggestion
    "documentation": "https://merchant-help.sezzle.com/hc/en-us/articles/16960335317908",
```

This page seems to be better.

File: entries/s/sezzle.com.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+  "Sezzle": {
+    "domain": "sezzle.com",
+    "tfa": [
+      "sms"
+    ],
+    "notes": "Only merchants can enable 2FA.",
```
Comment (hkamran80) [inline entries/s/sezzle.com.json]:
```suggestion
    "notes": "MFA is available for merchants only.",
```

**Review by hkamran80 — CHANGES_REQUESTED** [review]

**Comment by hkamran80** [comment]
I recommend submitting this to the [Passkeys Directory](https://github.com/2factorauth/passkeys) based on [this help page](https://merchant-help.sezzle.com/hc/en-us/articles/22057128759700).

**Review by hkamran80 — APPROVED** [review]

**Pull request merged**